contrastive_vi/module/total_contrastive_vi.py

```diff
@@ -686,14 +692,15 @@ def loss(
             background_losses["reconst_loss_protein"]
             + target_losses["reconst_loss_protein"]
         )
-        reconst_losses = dict(
-            reconst_loss_gene=reconst_loss_gene,
-            reconst_loss_protein=reconst_loss_protein,
+
+        wasserstein_loss = (
+            torch.norm(inference_outputs["background"]["qs_m"], dim=-1)**2
+            + torch.norm(torch.sqrt(inference_outputs["background"]["qs_v"]), dim=-1)**2
         )
 
-        kl_z = background_losses["kl_z"] + target_losses["kl_z"]
-        kl_s = target_losses["kl_s"]
-        kl_library_gene = (
+        kl_div_z = background_losses["kl_z"] + target_losses["kl_z"]
```

Comment:
Similar to contrastiveVI, I modified the names and the order of the loss terms to follow that of the scvi-tools totalVI implementation (https://github.com/scverse/scvi-tools/blob/master/scvi/module/_totalvae.py)